### DISCUSSION TITLE
Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by EmilPnunk** [mikeoliphant/neural-amp-modeler-lv2#72] [General]
Hi!

I'm very new to this coding business. I've been doing some simple programming (Industrial robots, PLC:s, that kind) at a former job, so I have a little bit of base knowledge atleast.
I'm also a guitarist, and recently I stumbled upon a used Mooer Preamp Live for a third of it's price at your typical musical store. And I was blown away, I finally found my sound, I really thought this was the holy grail I've been searching for all my life!
A couple of days later, and we have band practice. I'm a bit to excited about showing this new piece and the amazing guitar tone to my band mates, I plug in everything as fast as I can, but when the Mooer boots.. What the.. This is not the sounds I've spent all night tweaking.. It had somehow reset itself to it's factory settings! 
Fortunately, I remembered which ones of the amp- and pedal-simulations i used, so I quickly connected my phone via bluetooth to make something arbitrary that would atleast work for the rehearsal.

Then later when I come home, I'm of course a bit worried.. What if this happens right before a show? Disaster!
So I use my friend google to try to find out if this is a common problem. I don't find any stories about the thing resetting, but I find other horror stories about bricked devices, some people say they only last 1-2 years before they will be a doorstopper at best. 
So I send a message to the guy on facebook marketplace whom I bought the thing from, asking how long he owned. "About a year or so, and I also bought it second hand". Oh dear.. So I started to think about other solutions.. Should I buy a Kemper? A Helix? 
Then I remembered one of the things I wish I could improve with the Mooer. I would like to have more buttons, I don't like to double press a footswitch to access a pedal and so on. Then I also remembered stumbling upon some stories about Mooers shady business practices before, apparently they steal algorythms from other companies like there was no tomorrow. So I get the idea.. What if I could steal Mooers algorithms, and put them in a box of my own? And then I could put as many footswitches as I'd like on it?

So I spend the next days thinking, googling, dreaming, and eventually I stumbled upon your Youtubevideo, and here we are!
As I stated earlier, I don't have alot of experience coding, but I really think I could sort out building this! 
So now I'm guessing first step is to buy a Raspberry Pi. And then, finally, my question to you is:
Would this project work on a Raspberry Pi 5? It seems to me like it has way better specs, but I also read a comment by Elk Audio saying that they are currently not planning on porting their OS to the fifth gen raspberry since it uses a whole 'nother system.. They would have to rewrite alot of things. So I'm thinking maybe it's the same case with your project?

Also, sorry for wall of text. Hopefully you find it amusing!
And also, great work in making all of this! It's very very impressive!

**Comment by mikeoliphant** [comment]
You can absolutely do this sort of thing with a Raspberry Pi 5. In fact, I do it with a Pi 4.

This is the setup I use:

https://github.com/mikeoliphant/stompbox
